docs/usage/key-management.md

```diff
@@ -66,38 +66,21 @@ This command will:
 
 To import a keystore that was created via the ETH2.0 Deposit Launch Pad:
 
-```
-./lodestar account validator import --testnet medalla --directory <path to your launchpad keys>
+```bash
+./lodestar account validator import --testnet .$TESTNET_NAME --directory <path to your launchpad keys>
```

Comment:
Why a dot before the variable name TESTNET_NAME?

Reply:
woops!  just changed it